From 4db89e23190d1d1590d88df08056d327e651c94c Mon Sep 17 00:00:00 2001
From: Shoaib Meenai <smeenai@fb.com>
Date: Thu, 3 Feb 2022 13:39:54 -0800
Subject: [PATCH] [cmake] Increase -fms-compatibility-version in Windows
 toolchain file

Make it match LLVM's new minimum requirement (after https://reviews.llvm.org/D114639).
---
 llvm/cmake/platforms/WinMsvc.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/llvm/cmake/platforms/WinMsvc.cmake b/llvm/cmake/platforms/WinMsvc.cmake
index d30701a31858..ebb4da419e46 100644
--- a/llvm/cmake/platforms/WinMsvc.cmake
+++ b/llvm/cmake/platforms/WinMsvc.cmake
@@ -259,7 +259,7 @@ set(CROSS_TOOLCHAIN_FLAGS_NATIVE "${_CTF_NATIVE_DEFAULT}" CACHE STRING "")
 set(COMPILE_FLAGS
     -D_CRT_SECURE_NO_WARNINGS
     --target=${TRIPLE_ARCH}-windows-msvc
-    -fms-compatibility-version=19.14
+    -fms-compatibility-version=19.20
     -imsvc "${ATLMFC_INCLUDE}"
     -imsvc "${MSVC_INCLUDE}"
     -imsvc "${WINSDK_INCLUDE}/ucrt"
-- 
2.35.0.1.g829a698654

